css/style.css

```diff
@@ -1 +1,1091 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgb(13, 13, 17);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
+}
+
+/* Uneversal css */
+
+* {
+  padding: 0;
+  margin: 0;
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 28px;
+  color: rgba(255, 255, 255, 1);
+}
+
+a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 0 255px;
+}
+
+button,
+.btn-link,
+.btn-border {
+  padding: 12px 24px;
+  font-size: 16px;
+  line-height: 24px;
+  gap: 8px;
+  cursor: pointer;
+}
+
+span {
+  color: var(--btn-primary-color);
+}
+
+.secondary-heading {
+  font-size: 44px;
+  line-height: 56px;
+  text-align: center;
+}
+
+/* Header styles */
+
+header {
+  margin: 30px 0;
+}
+
+.logo-block {
+  font-size: 32px;
+  line-height: 42px;
+  gap: 16px;
+}
+
+.logo-block img {
+  width: 36px;
+  height: 40px;
+}
+
+/* Header navigation styles */
+
+.nav-menu-items {
+  gap: 32px;
+}
+
+.nav-menu-item a {
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 10%;
+}
+
+.dropdown img {
+  width: 24px;
+  height: 24px;
+}
+
+.dropdown-content a {
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 10%;
+}
+
+.btn-box {
+  gap: 20px;
+}
+
+.switch-mode {
+  width: 48px;
+  height: 48px;
+  margin: 0 auto;
+  padding: 12px;
+  border-radius: 41px;
+  background-color: var(--secondary-color);
+  border: none;
+}
+
+.switch-mode img {
+  width: 24px;
+  height: 24px;
+  background-color: transparent;
+}
+
+.header-link {
+  background-color: var(--secondary-color);
+}
+
+.header-link img {
+  width: 24px;
+  height: 24px;
+  background-color: transparent;
+}
+
+.btn-bgcolor {
+  border: none;
+  background-color: var(--btn-primary-color);
+  color: var(--secondary-color);
+}
+
+.header-btn img {
+  background-color: transparent;
+}
+
+/* Hero section */
+
+.hero-box {
+  padding-top: 141px;
+  /* add img bg  */
+  background-color: var(--secondary-color);
+}
+
+.hero-box * {
+  background-color: transparent;
+}
+
+.btn-bgcolor-hero {
+  border: none;
+  background-color: var(--btn-primary-color);
+  color: var(--secondary-color);
+}
+
+.hero-content {
+  width: 750px;
+}
+
+.hero-title,
+.hero-title span {
```

Comment:
.hero-title span does not need the same styles as .hero-title since text styles are inherited from the parent:

CSS inheritance automatically passes certain properties (like color, font-family, etc.) from parent elements to their children, so there’s no need to reapply the same styles unless you intend to override them.

[Learn more about CSS inheritance](https://developer.mozilla.org/en-US/docs/Web/CSS/inheritance).

Reply:
#fixed